doc/source/user_guide/variational_quantum_algorithms.rst

```diff
@@ -0,0 +1,30 @@
+Variational Quantum Algorithms with TensorLy-Quantum
+==============
```

Comment:
I think it will throw a syntax error if the '===' are not at least the same length as the text above.